include/ardpolyfills/Arduino.h

```diff
@@ -16,10 +16,40 @@
  *
  */
 
-#ifndef Arduino_h
-#define Arduino_h
+#ifndef SMARTCAR_EMUL_ARDUINO_HXX
+#define SMARTCAR_EMUL_ARDUINO_HXX
```

Comment:
Do not change the header guards name on this file, due to special compatibility with the AVR implementation